rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -524,6 +523,28 @@
         }
       }
     },
+    "client_id": {
```

Comment:
Client number?

Reply:
It's semantic... it's not always a number. It could be something like "BE-1234556". It's why I choose to use `_id` that seems more generic. @PascalRepond @JoelleDosimont  is it correct ?

Reply:
It's ok, you're right, somethimes the client "number" is not a number ;-)